src/ui/components/Transcript/Transcript.css

```diff
@@ -0,0 +1,27 @@
+.sidebar {
```

Comment:
How would you feel about using a CSS module here? I can't tell if we've decided that we're actually trying to go that direction for new style files or not?

Reply:
No strong opinion — would have inlined it but this was faster.